public/ticketmanager.php

```diff
@@ -294,102 +325,108 @@
+                    echo "<b><a href='$standardFilterURL" . ($ticketFilters & ~(1 << 7)) . "'>*RA Emulator</a></b> | ";
                 } else {
-                    echo "<a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters | (1 << 7)) . "'>RA Emulator</a> | ";
+                    echo "<a href='$standardFilterURL" . ($ticketFilters | (1 << 7)) . "'>RA Emulator</a> | ";
                 }
 
                 if ($rarchKnownTickets) {
-                    echo "<b><a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters & ~(1 << 8)) . "'>*RetroArch - Defined</a></b> | ";
+                    echo "<b><a href='$standardFilterURL" . ($ticketFilters & ~(1 << 8)) . "'>*RetroArch - Defined</a></b> | ";
                 } else {
-                    echo "<a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters | (1 << 8)) . "'>RetroArch - Defined</a> | ";
+                    echo "<a href='$standardFilterURL" . ($ticketFilters | (1 << 8)) . "'>RetroArch - Defined</a> | ";
                 }
 
                 if ($rarchUnknownTickets) {
-                    echo "<b><a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters & ~(1 << 9)) . "'>*RetroArch - Undefined</a></b> | ";
+                    echo "<b><a href='$standardFilterURL" . ($ticketFilters & ~(1 << 9)) . "'>*RetroArch - Undefined</a></b> | ";
                 } else {
-                    echo "<a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters | (1 << 9)) . "'>RetroArch - Undefined</a> | ";
+                    echo "<a href='$standardFilterURL" . ($ticketFilters | (1 << 9)) . "'>RetroArch - Undefined</a> | ";
                 }
 
                 if ($emulatorUnknownTickets) {
-                    echo "<b><a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters & ~(1 << 10)) . "'>*Emulator Unknown</a></b>";
+                    echo "<b><a href='$standardFilterURL" . ($ticketFilters & ~(1 << 10)) . "'>*Emulator Unknown</a></b>";
                 } else {
-                    echo "<a href='/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&f=$gamesTableFlag&t=" . ($ticketFilters | (1 << 10)) . "'>Emulator Unknown</a>";
+                    echo "<a href='$standardFilterURL" . ($ticketFilters | (1 << 10)) . "'>Emulator Unknown</a>";
                 }
                 echo "</div>";
 
                 // Core/Unofficial Filters - These filters are mutually exclusive
                 echo "<div>";
                 echo "<b>Achievement State:</b> ";
+                $gamesFilterUrl = '/ticketmanager.php?g=$gameIDGiven&u=$assignedToUser&p=$reportedByUser&r=$resolvedByUser&f=';
```

Comment:
this needs to be in double quotes to expand the variables

Reply:
You should just use `$standardFilterURL` here. It already contains `f=$gamesTableFlag`, which is the only thing you've extracted, and it immediately gets added back by the next couple of lines. The only difference is the explicit mapping from everything that's not 5 to 3. You could do that before constructing `$standardFilterUrl` so it's correct for all the other links too.

Reply:
I've tried to do this but I'm not seeing how it is possible since `$gamesTableFlag` needs to be set before `$standardFilterUrl` in order for it to take effect (which is fine for just changing !=5 to 3) but the `Core | Unofficial` filters have to have different values of `$gamesTableFlag`. All Urls need to follow `$standardFilterUrl` except for the opposite of the current `$gamesTableFlag` filter which can't be done without redefining `$standardFilterUrl` in one of the else statements which would then need to undo and go back to the original value right after. For now I will keep it as is and fix `$gamesFilterUrl` with double quotes.

Reply:
You're right. For the ones that change the value, you need to construct it with the alternate value, but for the ones where the value isn't changing, you could leverage the `$standardFilterUrl`. It's probably fine to reconstruct those too for readability.